tests/timezones/tests.py

```diff
@@ -7,6 +7,14 @@
 
 import pytz
 
+try:
```

Comment:
Note: This idiom gets repeated in a bunch of places, not sure if there's a "test support" module or something you want me to put it in. I frequently bury this sort of thing in a `_compat` module somewhere.

Reply:
I guess we could introduce `django.utils.zoneinfo` like we did with `simplejson` back in the time.

We could start deprecating the module when we drop support for Python 3.8.

Reply:
I wouldn't want anything user-facing here, since this is only ever used in the tests. If there's a common test utility module somewhere in the test suite where I can add functions like, "Get this IANA key in all supported time zone providers" and "skip this test if zoneinfo isn't available", it would make the PR a bit cleaner, but probably a lot of this stuff will get refactored for 4.0 anyway, so I don't think it's critical.

Reply:
right then I guess this import sequence is not too bad, it's already something we do in tests in a few other places when dealing with optional libraries.